front/public/locales/en/operationalStudies/manageTrainSchedule.json

```diff
@@ -53,6 +53,21 @@
   "infraLoading": "Infrastructure is loading…",
   "inputOPTrigrams": "Enter the trigrams of operational points you want to the path to go through, separated by a space.",
   "inputOPTrigramsExample": "Ex: LSN CIG CCS",
+  "incompatibleConstraints": {
+    "filters": {
+      "title": "Filter on constraint's type",
+      "enable": "Display constraints of type'{{name}}'",
+      "disable": "Hide constraints of type '{{name}}'"
+    },
+    "incompatible_electrification_ranges": "Incompatible electrification",
+    "incompatible_gauge_ranges": "Incompatible loading gauge",
+    "incompatible_signaling_system_ranges": "Incompatible signaling system",
+    "info": "Pathfinding failed due to some constraints. Here you will find the list of constraints that failed on the shortest path, with for each, the type of constraint, the position of the start of the failure zone relative to the start of the path, and its length.",
+    "gotoMap": "See on the map",
+    "seeConstraintsOnMap": "See constraints on the map",
```

Comment:
Here you use `See` but for the itinerary l.140, you use `View`. Should we harmonize that ?

And maybe it should say `all constraints`.

Reply:
It can't be "all constraints", because it focus on the not filtered constraints (ie. those that are displayed on the boxed). 
About the `view` vs `see` I'm agree that we need to choose one.